FHIR_README.md

```diff
@@ -0,0 +1,78 @@
+![img](./public/images/openemr-rest-api.png)
+
+### Overview
+
+Easy-to-use JSON-based REST API for OpenEMR FHIR. All code is done in classes and separate from the view to help with codebase modernization efforts. 
```

Comment:
Would be good to interllink the 2 docs (the API and this FHIR) here since they are very related to each other.

Reply:
Fixed